homeassistant/components/sensorpush/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "step": {
+      "user": {
+        "description": "Choose a device to setup",
+        "data": {
+          "address": "Device"
+        }
+      },
+      "bluetooth_confirm": {
+        "description": "Do you want to setup {name}?"
```

Comment:
Can we use a reference key to the Bluetooth integration?

Reply:
67acbba17020f5969d991c6997d232c5c58a02e0